code/web/src/modules/user/UserInfo.js

```diff
@@ -53,7 +53,7 @@ class UserInfo extends React.Component {
           this.props.messageShow('Image uploaded successfully.')
 
           let user = this.state
-          user.imageUrl = `/images/uploads/${response.data.file}`
+          user.imageUrl = `${response.data.file}`
```

Comment:
Nice job fixing this bug, glad we went over this part of the file and you caught it!